prefect_docker/projects/steps.py

```diff
@@ -18,20 +39,25 @@ class BuildDockerImageResult(TypedDict):
     The result of a `build_docker_image` step.
 
     Attributes:
-        image_name: The name and tag of the built image.
+        image_name: The name of the built image.
         image_tag: The tag of the built image.
+        image: The name and tag of the built image.
+        image_id: The ID of the built image.
     """
 
     image_name: str
     image_tag: str
+    image: str
+    image_id: str
 
 
 def build_docker_image(
     image_name: str,
     dockerfile: str = "Dockerfile",
     tag: Optional[str] = None,
-    push: bool = True,
+    push: bool = False,
```

Comment:
so the default behavior is changing here, correct?

Reply:
Yeah, I'm also planning on moving the pushing of images to a separate step since building & pushing has proved a stumbling block for users.